CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 ---
 
+## v1.2.2
+
+**Author**: Guillaume Mousnier.
+
+**Type**: Hotfix
+
+**Changes**:
+- Bugfix on join methods. They have now a correct behaviour. Thanks to @PaulMest
+- .joinOuter had a wrong behaviour, making the difference between dataframes. .outerJoin is now an alias for .fullJoin. The old behaviour can now be find into .diff.
```

Comment:
`.outerJoin` had a wrong behaviour. It  would show the difference between dataframes. `.outerJoin` is now an alias for `.fullJoin`. The old behaviour can now be found in `.diff`.